src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,39 @@
+package duke;
+
+/**
+ * Todo represents a todo for a task.
+ *
+ * @author Aaron Tan
+ */
+public class Todo extends Task {
```

Comment:
Hii, I like your well-organized code. Maybe you could use packages, (e.g. Putting Todo.java, deadline.java into a new Task Package) for better organization : )